### ISSUE TITLE
A question about symmetric encryption

### ISSUE THREAD

**Issue by RivenNero** [sh-dv/hat.sh#67]
Is the symmetric encryption algorithm of hat.sh XChaCha20-Poly1305-IETF or xchacha20-poly1305?

**Comment by covert-encryption** [comment]
Non-IETF, with 16 byte salt. You will find the code in https://github.com/sh-dv/hat.sh/blob/master/service-worker/sw.js#L212

**Comment by RivenNero** [comment]
OK
